apport/packaging_impl/apt_dpkg.py

```diff
@@ -1485,130 +1619,14 @@ def _search_contents(self, file, map_cachedir, release, arch):
-
-            contents_mapping = self._contents_mapping(map_cachedir, release, arch)
-            # if the mapping is empty build it
-            if not contents_mapping or len(contents_mapping) == 2:
-                self._contents_update = True
-            # if any of the Contents files were updated we need to update the
-            # map because the ordering in which is created is important
-            if self._contents_update:
-                with gzip.open(f"{contents_filename}", "rb") as contents:
-                    line_num = 0
-                    for line in contents:
-                        line_num += 1
-                        # the first 32 lines are descriptive only for these
-                        # releases
-                        if (
-                            pocket == ""
-                            and release in {"trusty", "xenial"}
-                            and line_num < 33
-                        ):
-                            continue
-                        path = line.split()[0]
-                        if path.split(b"/")[0] == b"usr":
-                            if path.split(b"/")[1] not in (
-                                b"lib",
-                                b"libexec",
-                                b"libx32",
-                                b"bin",
-                                b"sbin",
-                                b"share",
-                                b"games",
-                                b"Brother",
-                            ):
-                                continue
-                            if path.split(b"/")[1] == b"share" and path.split(b"/")[
-                                2
-                            ] in {
-                                b"doc",
-                                b"icons",
-                                b"man",
-                                b"texlive",
-                                b"gocode",
-                                b"locale",
-                                b"help",
-                            }:
-                                continue
-                            package = line.split()[-1].split(b",")[0].split(b"/")[-1]
-                        elif path.split(b"/")[0] in {b"lib", b"bin", b"sbin", b"etc"}:
-                            package = line.split()[-1].split(b",")[0].split(b"/")[-1]
-                        else:
-                            continue
-                        if path in contents_mapping:
-                            if package == contents_mapping[path]:
-                                continue
-                            # if the package was updated use the update
-                            # b/c everyone should have packages from
-                            # -updates and -security installed
-                        contents_mapping[path] = package
+        contents_mapping = self._get_contents_mapping(map_cachedir, release, arch)
```

Comment:
```suggestion
        file2pkg = self._get_contents_mapping(map_cachedir, release, arch)
```

We don't really care that it comes from a Contents file once we have the mapping, and that name makes its contents (heh!) explicit.

Reply:
Renamed and also adjusted some of the helper methods to use that name.